### ISSUE TITLE
Current beta crashes if I go to settings

### ISSUE THREAD

**Issue by fraternl** [SubtitleEdit/subtitleedit#8105]
Current beta crashes if I go to settings.
even if I extract it to "C:\Program Files\Subtitle Edit" after emptying it first.

<img width="572" alt="image" src="https://github.com/SubtitleEdit/subtitleedit/assets/8692856/6ab2daa2-fbd6-495f-a0fb-d5e2d1305636">

**Comment by niksedk** [comment]
The beta versions are portable versions. Portable versions should not be installed under `C:\Program files` which should be only used for installers.

The beta should be unpacked to something like `C:\Tools\SE`.

**Comment by fraternl** [comment]
It indeed doesn't crash then...
But I extracted a beta-version quite often over my current version and never bumped into this.

Is there a reason you are aware of that it crashes now and not before?

**Comment by niksedk** [comment]
Yes, thx. There was a reason for the crash... I had not included a default image for the new "Open video" toolbar button.

Fixed in latest beta: https://github.com/SubtitleEdit/subtitleedit/releases/download/4.0.4/SubtitleEditBeta.zip
